gulpfile.js

```diff
@@ -457,29 +471,33 @@ gulp.task('release', ['apps', 'clean-release'], function () {
     var platforms = getPlatforms();
     console.log('Packing release.');
 
+    var merged = mergeStream();
+
     if (platforms.indexOf('chromeos') !== -1) {
-        release_chromeos();
+        merged.add(release_chromeos());
     }
 
     if (platforms.indexOf('linux64') !== -1) {
-        release('linux64');
+        merged.add(release('linux64'));
     }
 
     if (platforms.indexOf('linux32') !== -1) {
-        release('linux32');
+        merged.add(release('linux32'));
     }
         
     if (platforms.indexOf('osx64') !== -1) {
-        release_osx64();
+        merged.add(release_osx64());
     }
 
     if (platforms.indexOf('win32') !== -1) {
-        release_win('win32');
+        merged.add(releaseWin('win32'));
     }
     
     if (platforms.indexOf('win64') !== -1) {
-        release_win('win64');
+        merged.add(releaseWin('win64'));
```

Comment:
The `releaseWin` is now `release_win` and is a sync task. It does not return a `stream` so it is incorrect to add it to the `merge-stream`. The original way is correct: `release_win('win64');`. The same for `win32`.

Reply:
Thanks, I didn't know this and it probably got changed with rebasing.

Reply:
As the windows release tasks sync tasks, this means that windows releases can't be build with each other or one of the other plaform at the same time?

Reply:
They can be build in the same command, but not in parallel (async). I've a hack to let them be build in parallel, I will push the PR after this PR is merged ;)